### PR TITLE
The variable names prefix and suffix are incorrectly identified in th…

### LINKED ISSUE #907: Parameter variables incorrectly identified as suffix for prefix content and vice versa
Recently in review of some code, found that the functions zen_get_show_product_switch and zen_get_show_product_switch_name in both the catalog and admin side identify the parameter variable $prefix but it is used as a suffix and the parameter variable $suffix as a prefix.  This simply reverses the assignment(s) so that help offered like on the current development site (doesn't show the code of the declared function) is better understood.

### PR DESCRIPTION
**Pull request by mc12345678**
Fixes #907.  Swaps variable names $suffix and $prefix in two functions each on the catalog and admin side.

### CODE REVIEW

File: includes/functions/functions_lookups.php [hunk 1]
```diff
@@ -676,7 +676,8 @@ function zen_get_products_allow_add_to_cart($lookup) {
 /*
  * Look up SHOW_XXX_INFO switch for product ID and product type
  */
-    function zen_get_show_product_switch_name($lookup, $field, $suffix= 'SHOW_', $prefix= '_INFO', $field_prefix= '_', $field_suffix='') {
+    function zen_get_show_product_switch_name($lookup, $field, $prefix= 'SHOW_', $suffix= '_INFO', $field_prefix= '_', 
+                                              $field_suffix='') {
```
Comment (drbyte) [inline includes/functions/functions_lookups.php]: Other than adding a line-break to wrap to the next line in 4 places in this file, was there anything else changed in this file?

Comment (drbyte) [inline includes/functions/functions_lookups.php]: Oh, duh.  github highlighting was wonky. I see it now.

**Comment by drbyte** [comment]
👍

**Comment by zcwilt** [comment]
:+1:

**Pull request merged**